packages/features/sales-escrows/modals/StartEscrowModal.tsx

```diff
@@ -105,8 +114,15 @@ export function StartEscrowModal({
     setEnteredAmount(enteredAmount);
 
     let validationMsg = validateTokenAmount(enteredAmount, token.decimals);
+
+    const fee = toLargerUnit(token.fee, token.decimals);
+    const amount = toBigNumber(enteredAmount);
```

Comment:
This is still called before validationMsg is checked. If validationMsg contains and error message, that means that enteredAmount can not be converted to a number and will throw an error. You can modify the code to look like validateForm. There if there is a validationMsg, it return false. After that, you can safely convert enteredAmount to a number.